src/platform/tls_openssl.c

```diff
@@ -3144,7 +3184,7 @@ CxPlatTlsProcessData(
             }
         }
 
-        if (TlsContext->State->WriteKey == QUIC_PACKET_KEY_1_RTT) {
+        if (TlsContext->State->WriteKey == QUIC_PACKET_KEY_1_RTT && AData->SecretSet[3][0].Secret != NULL) {
```

Comment:
Can we use some constants or enums instead of magic numbers?  For example, I know `direction` is either `0` or `1` but from just a glance, I don't remember which is receive and which is send.